File: app/tklaundry_app/lib/features/expend/domain/expend.dart
class Expend {
  const Expend({
    required this.idx,
    required this.expendDate,
    required this.expendCode,
    required this.cost,
    required this.remark,
  });

  final int idx;
  final String expendDate;
  final String expendCode;
  final int cost;
  final String remark;

  factory Expend.fromJson(Map<String, dynamic> json) {
    return Expend(
      idx: json['idx'] as int? ?? 0,
      expendDate: json['expendDate'] as String? ?? '',
      expendCode: json['expendCode'] as String? ?? '',
      cost: json['cost'] as int? ?? 0,
      remark: json['remark'] as String? ?? '',
    );
  }
}

File: backend/tklaundry-api/src/main/java/com/tklaundry/api/sales/expend/model/Expend.java
package com.tklaundry.api.sales.expend.model;

import java.time.LocalDate;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.tklaundry.api.common.model.BaseEntity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@JsonInclude(JsonInclude.Include.NON_NULL)
@Getter
@Setter
@SuperBuilder(toBuilder = true)
@NoArgsConstructor
@AllArgsConstructor
public class Expend extends BaseEntity {

	private Integer idx;
	private LocalDate expendDate;
	private String expendCode;
	private Integer cost;
	private String remark;

}
